dallinger/recruiters.py

```diff
@@ -171,8 +200,19 @@ def close_recruitment(self):
         """
         logger.info("Close recruitment.")
 
+    def _extract_blacklist_from_config(self):
+        # At some point we'll support lists, so all service code supports them,
+        # but the config system only supports strings for now, so we convert:
+        blacklist = self.config.get('blacklist')
+        if blacklist:
+            blacklist = tuple([item.strip() for item in blacklist.split(',')])
+        else:
+            blacklist = ()
+
+        return blacklist
```

Comment:
It makes me uneasy that this is something that the developer looking up the config setting has to remember to do. I think we should extend the configuration system so that a particular setting can be associated a function that is responsible for parsing it, which would happen automatically as part of `Configuration.extend`.